Rakefile

```diff
@@ -9,6 +9,32 @@ task :test do
   sh "python -m tests.benchmark"
 end
 
+desc 'CI dependent task; tasks in parallel'
+task:scalable_test do
```

Comment:
eventually we should kill the original `rake test` task, but let's keep it around while we make sure there are no regressions